notebooks/official/feature_store/sdk-feature-store-pandas.ipynb

```diff
@@ -30,15 +30,23 @@
       },
```

Comment:
<p>Suggest: "This notebook introduces Pandas support for Vertex AI Feature Store by using the Vertex AI SDK for Python. For prerequisites and an introduction to Vertex AI Feature Store, see the Vertex AI SDK for Python Feature Store notebook." &lt;-- maybe we should review that notebook first to make it official.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>